addons/crm/controllers/main.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import werkzeug
+from werkzeug import url_encode
+
+from odoo.addons.mail.controllers.main import MailController
+from odoo import http
+from odoo.http import request
+
+
+class HrHolidaysController(http.Controller):
```

Comment:
Sometimes I have a feeling that someone spies me :) . Anyway this is just a WIP. But you are right and I modified it. Copy and paste magic.